src/services/notifications/index.ts

```diff
@@ -67,18 +59,22 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
       })
 
       // Notify the recipient (via GraphQL subscription if any)
-      const accountUpdatedEventName = accountUpdateEvent(account.id)
-
-      pubsub.publish(accountUpdatedEventName, {
-        transaction: {
-          walletId,
-          txNotificationType: type,
-          amount: sats,
-          txHash,
-          displayCurrencyPerSat,
+      const accountUpdatedTrigger = customPubSubTrigger({
+        event: PubSubDefaultTriggers.AccountUpdate,
+        suffix: account.id,
+      })
+      pubsub.publish({
+        trigger: accountUpdatedTrigger,
+        payload: {
+          transaction: {
+            walletId,
+            txNotificationType: type,
+            amount: sats,
+            txHash,
+            displayCurrencyPerSat,
+          },
         },
       })
-      return
```

Comment:
Could this method me a candidate for "default to returning `true` instead of `void` so we always have explicit returns"?